backend/pvm.pm

```diff
@@ -28,7 +28,22 @@ sub new ($class) {
     $self->{pvmctl} = $ENV{PVMCTL} // '/usr/bin/pvmctl';
     $self->{masterlpar} = substr(_masterlpar, 0, -1);
     die "pvmctl not found" unless -x $self->{pvmctl};
-    bmwqemu::fctwarn 'DEPRECATED: backend::pvm is unsupported and planned to be removed from os-autoinst eventually';
+    my $backend = 'PVM';
+    my $deprecation_message = <<"EOF";
+DEPRECATED: 'backend::$backend' is unsupported and planned to be
+removed from os-autoinst eventually. If the backend is still needed please
+report an issue on https://github.com/os-autoinst/os-autoinst . This message
+can be temporarily turned into a warning by setting the environment variable
+'OS_AUTOINST_NO_DEPRECATE_BACKEND_$backend' or the os-autoinst variable
+'NO_DEPRECATE_BACKEND_$backend'
+EOF
+    if ($bmwqemu::vars{"NO_DEPRECATE_BACKEND_$backend"} ||
+        $ENV{"OS_AUTOINST_NO_DEPRECATE_BACKEND_$backend"}) {
+        log::fctwarn $deprecation_message;
+    }
+    else {
+        die $deprecation_message;
+    }
```

Comment:
Maybe implement it as a method in the baseclass, and call it from here passing the backend's name?